Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+  Scanner input = new Scanner(System.in);
+
+  // ask the user for two integers
+  System.out.println("Please enter an integer:");
+  int a = input.nextInt();
+  System.out.println("Please enter another integer:");
+  int b = input.nextInt();
+
```

Comment:
a and b are normally poor choices for names. Instead, use something like num1 and num2.